fetch.bs

```diff
@@ -3881,8 +3889,8 @@ the request.
  <var>crossOriginIsolatedCapability</var>.
 
  <li><p>If <var>request</var>'s <a for=request>body</a> is a <a for=/>byte sequence</a>, then set
- <var>request</var>'s <a for=request>body</a> to the first return value of
- <a for=BodyInit>safely extracting</a> <var>request</var>'s <a for=request>body</a>.
+ <var>request</var>'s <a for=request>body</a> to the <a for="body with type">body</a> in the result
```

Comment:
I think it should be "of the result" (applies several times), but not a 100%.

Reply:
I switched it between "of" and "in" a couple of times before submitting the PR. I am not sure either. It's kind of a weird sentence construction.